chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/TestTask.java

```diff
@@ -31,7 +32,7 @@
 @XmlType(namespace = "http://chpl.healthit.gov/listings")
 @XmlAccessorType(XmlAccessType.FIELD)
 @JsonIgnoreProperties(ignoreUnknown = true)
-@Builder
+@Builder(toBuilder = true)
```

Comment:
What does the `toBuilder = true` do?

Reply:
It adds a method "toBuilder" to the class so you can take any instance of TestTask and create a copy of it as a builder class with all of its values.  So for example if I parsed upfront a bunch of test tasks and their details and want to make a copy of that test task and it's ten thousand fields already filled in with data to put it into our Sed object and associate it with the criterion/a with which is belongs I could do that with less code. I think you might do something like this with a copy constructor if you didn't have lombok and builder.

Reply:
Cool. Good to know